manifests/snmpv3_user.pp

```diff
@@ -39,8 +39,8 @@
   }
 
   $cmd = $privpass ? {
-    undef   => "createUser ${title} ${authtype} \\\"${authpass}\\\"",
-    default => "createUser ${title} ${authtype} \\\"${authpass}\\\" ${privtype} \\\"${privpass}\\\""
+    undef   => "createUser ${title} ${authtype} \"${authpass}\"",
```

Comment:
Do we actually need double quotes around those strings? Can't we use single quotes?

Reply:
Maybe. I will do some tests when I get back into office.

Reply:
It seems that single quotes would also work.

Maybe the quoting should be reviewed. Quotes are only required when the password contains spaces. On the other hand passwords may also include quotes which then need to be escaped (not currently implemented). We could drop the quotes in Puppet and forget about escaping if we would just prohibit passwords with spaces.